src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -20,7 +28,7 @@ const Root = styled.View`
 const UpperSection = styled.View`
   width: 140%;
   height: 45%;
-  background-color: ${prop => prop.theme.colors.primary};
+  background-color: ${ExerciseUnlocked ? prop => prop.theme.colors.correct : prop => prop.theme.colors.primary};
```

Comment:
You move `props =>` to the beginning, makes it a little simler: `${prop => (ExerciseUnlocked ? prop.theme.colors.correct : prop.theme.colors.primary)};`

Same below for message.